docs/spelling_wordlist

```diff
@@ -209,6 +209,7 @@ hashable
 hasher
 hashers
 headerlist
+hiredis
```

Comment:
Do we need this, maybe we should use `hiredis` instead? (ummm... that's surround it in double backticks in the docs)